### ISSUE TITLE
Current Qt 6.5 Known Issues

### ISSUE THREAD

**Issue by Silarn** [ModOrganizer2/modorganizer#1866]
6.5.0:
* Possible QSS icon display issue related to hover state

6.5.1:
* Having a tab selected prior to rendering the tab window, such as with window geometry saved settings, will cause earlier tabs to fail to render. The only way to fix it (temporarily) is to delete the saved tab index.

6.5.2:
* Alternating QTreeView rows fail to display custom QSS icons
* Context menus always render on the primary monitor regardless of the mouse location
* Reported display issues with high DPI monitors

**Comment by Holt59** [comment]
@Silarn Did you try building with Qt 6.5.3 or Qt 6.6 (beta)?

**Comment by Silarn** [comment]
No, since they are not officially released. But I guess 6.5.3 will be out within a week.

**Comment by Holt59** [comment]
I will try building with Qt 6.6 and see if it fixes some of the bugs - Not to do a release or even a beta build with it, just to see what to expect for the next months or so from Qt.

**Comment by Holt59** [comment]
This is what I can say from Qt 6.6 current build:

> 6.5.0:
> 
>     * Possible QSS icon display issue related to hover state

If think this is fixed, if that was about the kind of "duplicated" icon. The margin issue is still present (when you hover, the text is moved right when there is an icon or a checkbox).

> 6.5.1:
> 
>     * Having a tab selected prior to rendering the tab window, such as with window geometry saved settings, will cause earlier tabs to fail to render. The only way to fix it (temporarily) is to delete the saved tab index.

I think this is fixed, although I did not experience this bug before so maybe I am not checking the right thing. I tried opening/closing MO2 with various tabs selected and then opening other tabs and everything looks fine.

> 
> 6.5.2:
> 
>     * Alternating QTreeView rows fail to display custom QSS icons

That's still the case...

>     * Context menus always render on the primary monitor regardless of the mouse location

Half-fixed, the context menu open on the right display but seems to use the scaling of the primary monitor. I have a 4K primary monitor and a 2K secondary, and the context menu on the secondary are oversized and mostly unusable.
 
>     * Reported display issues with high DPI monitors

I don't what these issues are so cannot say.

**Comment by Silarn** [comment]
Yeah I think the earlier issues were resolved by 6.5.2 but they introduced the alternating icon and context menu problems. Sounds like those are still problems.

I also can't comment on the display issues, there were just a couple people that mentioned having scanning problems.

**Comment by Silarn** [comment]
Just FYI the place I noticed it was in the settings menu. For 6.5.1.

**Comment by ashemedai** [comment]
Qt 6.6.1 seems to cause the ModInfoDialog window to go maximum size. Haven't been able to track down if this is something in the code base or a bug in 6.6.1.
6.6.1 does fix the alternating icon issue though.

**Comment by pritster5** [comment]
Is it at all possible to zoom out the UI in MO2? I just updated to the latest version and everything is bigger. I'd like to zoom out.
